assets/crd/0000_01_route.crd.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Where is this file copied from (for the rebase script), please?

Reply:
now from https://github.com/openshift/api/pull/1228/files#diff-6f2a93a87b2c5d86ab777d70320bba54ba8395569f69dc4b4446ac9c3cb76480

Eventually from the openshift/api repo once the PR is green.

Reply:
Now present in https://github.com/openshift/api/blob/master/route/v1/route.crd.yaml